packages/server/database/migrations/20200530185318-addMeetingIdToAgendaItem.ts

```diff
@@ -0,0 +1,23 @@
+import {R} from 'rethinkdb-ts'
+
+export const up = async function (r: R) {
+  try {
+    await r
+      .table('AgendaItem')
+      .update({meetingId: ''})
```

Comment:
curious, why add a meetingId field to all records?

Reply:
it seems that currently, (as a free user) there's no way to go back to a meeting that has ended to see old discussion threads. with that, it seems like we'd only need to backfill `meetingId` on `agendaItem` for any open active action meetings in production. 

however, it seems possible this product behavior may change in the future. i'm thinking of slack and times when i've needed to search through old archives to find something specific. to enable that behavior, we'd need `meetingId` on all records. 

i'm not too familiar with the product roadmap yet, so i don't have intel on that front. on the engineering side, it seems cheap to populate an `id` field. I would argue that it's more future proof to backfill it on all records. lmk your thoughts